Porting/checkAUTHORS.pl

```diff
@@ -900,6 +900,7 @@ sub _raw_address {
 eggert\100twinsun.com                   eggert\100sea.sm.unisys.com
 etj\100cpan.org                         mohawk2\100users.noreply.github.com
 
+felipe\100felipegasper.com              fgasper\100users.noreply.github.com
```

Comment:
Why is this necessary?

Reply:
Because otherwise the authors test fails. :(

Reply:
None of the commits is assigned to that email address, that makes no sense.

Reply:
I'll pull it out, but I suspect the authors test will fail (again).

Reply:
4c1e2e1d31d9f9ceab9a194e1e9b5f2bc30798f1 should have fixed that issue, so this addition shouldn't be necessary anymore

Reply:
Indeed; I’ve pulled it out.